statsmodels/tsa/automatic/forecast.py

```diff
@@ -0,0 +1,120 @@
+import numpy as np
+import pandas as pd
+from scipy.stats import norm
+from statsmodels.tools.decorators import cache_readonly
+
+
+class Forecast:
```

Comment:
We still support Python 2, so we need to force this to be a new style class with `class Forecast(object):`

Reply:
Thanks for letting me know, I have made the changes